lib/redcord/migration/ttl.rb

```diff
@@ -2,10 +2,22 @@
 module Redcord::Migration::TTL
   extend T::Sig
 
+  sig { params(model: T.class_of(Redcord::Base)).returns(T.untyped) }
+  def _get_ttl(model)
+    model.class_variable_get(:@@ttl) || -1
+  end
+
   # This won't change ttl until we call update on a record
   sig { params(model: T.class_of(Redcord::Base)).void }
   def change_ttl_passive(model)
-    ttl = model.class_variable_get(:@@ttl)
-    model.redis.set("#{model.model_key}:ttl", ttl ? ttl : -1)
+    model.redis.set("#{model.model_key}:ttl", _get_ttl(model))
+  end
+
+  sig { params(model: T.class_of(Redcord::Base)).void }
+  def change_ttl_active(model)
+    change_ttl_passive(model)
+    model.redis.keys("#{model.model_key}:id:*").map do |key|
```

Comment:
https://redis.io/commands/keys I'm not sure we should use the keys operation in production 

Alternatively we could do a scan with a pattern match? 
`redis.call('scan', cursor, 'match', 'model_key:id:*')`

Reply:
Maybe we can write a lua script which does a scan, and calls expire on each model key returned from scan. We can loop through the cursor on the client side

https://github.com/chanzuckerberg/redcord/blob/kho/vacuum/lib/redcord/server_scripts.rb#L104 this is still very much a work in progress for the vacuum logic, but I was planning on doing something similar here.

Reply:
> lternatively we could do a scan with a pattern match?

great idea! we should avoid blocking as much as possible.

For the same reason, we probably should not use a script here to block less